release_notes.md

```diff
@@ -1,3 +1,41 @@
+***********************************
+# Release Notes for OpenMDAO 3.11.0
+
+August 03, 2021
+
+OpenMDAO 3.11.0 introduces SemiStructuredMetaModelComp as well as some bug fixes and performance improvements.
+It fixes the known issue from 3.10.0 in which the derivatives of serial outputs wrt distributed inputs were broken.
+
+## New Deprecations
+
+- None
+
+## Backwards Incompatible API Changes:
+
+- None
+- 
+## Backwards Incompatible NON-API Changes:
+
+- None
+
+## New Features:
+
+- New MetaModelSemiStructuredComp component that can interpolate on "semi-structured" data. [#2185](https://github.com/OpenMDAO/OpenMDAO/pull/2185)
+
+## Bug Fixes:
+
+- Fixed deprecation of 'value' in get_io_metadata(). [#2191](https://github.com/OpenMDAO/OpenMDAO/pull/2191)
+- Improved error message for global shape errors. [#2192](https://github.com/OpenMDAO/OpenMDAO/pull/2192)
+- Use backward difference for check options on structured metamodel with slinear. [#2195](https://github.com/OpenMDAO/OpenMDAO/pull/2195)
```

Comment:
for 2195, i would move this to Miscellaneous section, because the user won't ever experience this unless they set `_no_check_partials` to False on their metamodel, which is a bit advanced.

Also, reword to "The MetaModelStructuredComp now uses backward difference for fd checks when the method is 'slinear' so that the finite difference step direction aligns with the bin bracketing."